src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts

```diff
@@ -872,6 +876,39 @@ export class ManageExtensionAction extends Action {
 	}
 }
 
+export class ExtensionInfoAction extends Action implements IExtensionAction {
+
+	static readonly ID = 'extensions.extensionInfo';
+	static LABEL = localize('extensionInfoAction', "Copy Extension info to clipboard");
```

Comment:
The `to clipboard` here is unnecessary. We have other copy actions in the product where it is assumed that the user knows that the text is copied to clipboard.

How about `Copy Extension information`?

Reply:
Done